simplex.py

```diff
@@ -276,7 +269,7 @@ def run(self, r_data=None):
                 data = calculate.main(self.args_ff)
                 con_ff.score = compare.compare_data(r_data, data)
                 opt.pretty_ff_results(con_ff)
-                if con_ff.score < self.new_ffs[-2].score:
```

Comment:
I too always found this odd, but that's what was in the old code, and actually your 1998 paper on Q2MM. 

> If pk (contracted) is better than pn (2nd worst), replace pw (worst) by pk, continue with next cycle. If not, do a massive contraction.